modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsaMPCv2.ts

```diff
@@ -318,21 +322,30 @@ export class EcdsaMPCv2Utils extends BaseEcdsaUtils {
     participantIndex: MPCv2Party,
     commonKeychain: string,
     privateMaterial?: Buffer,
+    reducedPrivateMaterial?: Buffer,
     passphrase?: string,
     originalPasscodeEncryptionCode?: string
```

Comment:
Nit, chaining optionals like this gets messy. Better to have an object with optional fields.